neo4j-datasource-plugin/src/QueryEditor.tsx

```diff
@@ -1,17 +1,43 @@
 import React, { PureComponent } from 'react';
-import { QueryField } from '@grafana/ui';
-import { QueryEditorProps } from '@grafana/data';
+import { InlineFieldRow, InlineFormLabel, QueryField, Select } from '@grafana/ui';
+import { QueryEditorProps, SelectableValue } from '@grafana/data';
 import { DataSource } from './datasource';
-import { MyDataSourceOptions, MyQuery } from './types';
+import { MyDataSourceOptions, MyQuery, Format } from './types';
 
 type Props = QueryEditorProps<DataSource, MyQuery, MyDataSourceOptions>;
 
+const Formats = [
+  {
+    label: 'Table',
+    value: Format.Table,
+    description: 'Table View',
+  },
+  {
+    label: 'Node Graph',
+    value: Format.NodeGraph,
+    description: 'Node Graph View',
+  },
+] as Array<SelectableValue<Format>>;
+
 export class QueryEditor extends PureComponent<Props> {
   onCypherQueryChange = (value: string) => {
     const { onChange, query } = this.props;
     onChange({ ...query, cypherQuery: value });
   };
 
+  onFormatChanged = (selected: SelectableValue<Format>) => {
+    const { onChange, query, onRunQuery } = this.props;
+    onChange({ ...query, Format: selected.value || Format.Table });
+    onRunQuery();
+  };
+
+  resolveFormat = (value: string | undefined) => {
+    if (value === Format.Table) {
+      return Formats[0];
+    }
+    return Formats[1];
+  };
```

Comment:
We should make Table the default, because syntax is not straightforward to easily see data 

```
  resolveFormat = (value: string | undefined) => {
    if (value === Format.NodeGraph) {
      return Formats[1];
    }
    return Formats[0];
  };
```

Reply:
You're right.

Reply:
I applied the proposed changes.